utils/benchmarking/eventing/relay/main.go

```diff
@@ -0,0 +1,124 @@
+// MIT License
+//
+// Copyright (c) 2021 Mert Bora Alper and EASE lab
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"strconv"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/ease-lab/vhive/utils/benchmarking/eventing/proto"
+)
+
+const DatabaseAddress = "timeseriesdb.default.192.168.1.240.sslip.io:80"
+
+func UnmarshalVHiveMetadata(s string) *proto.VHiveMetadata {
+	var j struct {
+		Id        string `json:"Id"`
+		InvokedOn string `json:"InvokedOn"`
+	}
+	if err := json.Unmarshal([]byte(s), &j); err != nil {
+		log.Fatalf("failed to unmarshal vhivemetadata: %s", err)
+	}
+	invokedOnT, err := time.Parse("2006-01-02T15:04:05-0700", j.InvokedOn)
```

Comment:
why this constant?

Reply:
Go uses a specific date to format timestamps (where other languages use printf-style formatters such as `%Y-%m-%dT%H:%M:%S`). This is [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard, but I will switch to [`time.RFC3339Nano`](https://golang.org/pkg/time/#pkg-constants) instead for better precision.